pkg/nftables/firewall.go

```diff
@@ -235,7 +236,7 @@ func (f *Firewall) validate(file string) error {
 }
 
 func (f *Firewall) reconcileIfaceAddresses() error {
-	var errors *multierror.Error
+	var errs *multierror.Error
```

Comment:
i would prefer to completely get rid of multierr and use errors.Join at the end from the stdlib